CONTRIBUTING.adoc

```diff
@@ -85,7 +85,7 @@ $ sudo dnf install git docker-latest
 ----
 In order to do builds locally, install the following build dependencies:
 ----
-$ sudo dnf install golang golang-race make gcc zip mercurial krb5-devel bsdtar bc rsync bind-utils file jq tito createrepo openssl gpgme gpgme-devel libassuan libassuan-devel
+$ sudo dnf install golang golang-race make gcc zip mercurial krb5-devel bsdtar bc rsync bind-utils file jq tito createrepo openssl gpgme gpgme-devel libassuan libassuan-devel protobuf-compiler
```

Comment:
We don't need it "to do builds locally" , do we? Maybe we should mention it separately as optional dependency?

Also, I remember that there is another way for updating generated protobuf definitions that don't require to install this but it uses docker instead. I couldn't find an e-mail from @smarterclayton about this...